addons/account_edi/models/account_edi.py

```diff
@@ -0,0 +1,251 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api
+from odoo.exceptions import UserError
+from odoo.tools.pdf import OdooPdfFileReader, OdooPdfFileWriter
+
+from lxml import etree
+import base64
+import io
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountEdi(models.Model):
+    _name = 'account.edi'
+    _description = 'EDI format'
+
+    name = fields.Char(index=True)
+    code = fields.Char()
+    export_invoice = fields.Selection(selection=[('embed_to_pdf', 'Embed in PDF'), ('separated_attachment', 'Attachment'), ('not_supported', 'Not Supported')], 
```

Comment:
I didn't see this entry in the selection. This is the reason why you should'nt exceed 140 chars. This is more readable to me:
```
 export_invoice = fields.Selection(selection=[
  ('embed_to_pdf', 'Embed in PDF'),
  ('separated_attachment', 'Attachment'),
  ('not_supported', 'Not Supported'),
], default='embed_to_pdf')

```

Reply:
Some EDI only support import and not export.

For example, Italy, the export caused many problems so we chose not to adapt it now. Instead it is still managed internally. But we still need to adapt the import otherwise it would have broken the feature.

Reply:
I added a comment to make that  clear.